lean-code-gate-v3/tests/test_lean_code_gate.py

```
@@ -403,6 +403,78 @@ def test_reimplemented_dedupe_loop_fails_as_high_confidence_reuse() -> None:
+            encoding="utf-8",
+        )
+        git(repo, "add", ".")
+        git(repo, "commit", "--no-gpg-sign", "-m", "seed signature")
+        (repo / "src" / "wrapper.py").write_text(
+            "from src.signature import compute_html_signature\n"
+            "def wrap_payload(blob, fn=compute_html_signature()):\n"
+            "    digest = hashlib.sha256(blob).hexdigest()\n"
+            "    return fn or digest[:16]\n",
+            encoding="utf-8",
+        )
+        code, data = check_json(repo)
```

Ruff (0.15.12)

[warning] Unpacked variable `code` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)